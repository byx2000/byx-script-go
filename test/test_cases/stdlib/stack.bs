import stack

var s = Stack()
s.push(1)
s.push(2)
s.push(3)
println(s.size(), s.isEmpty())

println(s.pop())
println(s.pop())
println(s.size(), s.isEmpty())

s.push(4)
s.push(5)
println(s.top())
println(s.size())

println(s.pop())
println(s.pop())
println(s.pop())
println(s.size(), s.isEmpty())
