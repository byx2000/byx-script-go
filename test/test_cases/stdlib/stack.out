3 false
3
2
1 false
5
3
5
4
1
0 true